test/test-priority.sh

```
@@ -0,0 +1,214 @@
+RED='\033[0;31m'
+YELLOW='\033[1;33m'
+NC='\033[0m' # No Color
+
+# Setup
+echo "🧪 Shelly Priority Tests"
+echo "========================"
+echo ""
+
+rm -rf $TEST_DIR
+mkdir -p $TEST_DIR
+cd $TEST_DIR
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)